Utilities/ocvalidate/Checklist.md

```diff
@@ -0,0 +1,65 @@
+ocvalidate Checklist
+=====================
+
+As of commit <TODO>, ocvalidate performs the following checks:
+
+## ACPI
+- [ ] Add->Entry[N]->Path: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
+- [ ] Add->Entry[N]->Path: `.dsl` filename suffix is not accepted.
+- [ ] Add->Entry[N]->Path: If a customised DSDT is added, remind to enable RebaseRegions in Quirks.
+- [ ] Add->Entry[N]->Comment: Only ASCII printable characters are accepted.
```

Comment:
Maybe put it to global section? Like: "For all comments only ASCII printable characters are accepted." to reduce the bloat?

Reply:
Done.